openwisp_controller/urls.py

```diff
@@ -34,7 +34,14 @@
     },
 ]
 
-urlpatterns = []
+urlpatterns = [
+    url(
+        '',
+        include(
+            ('openwisp_controller.admin_urls', 'config_admin'), namespace='config_admin'
+        ),
+    )
+]
```

Comment:
if you use the previous suggestion this may not be needed

Reply:
For removing this, I had to add `url(r'', include(('openwisp_controller.config.urls', 'config'),namespace='config')),` to register this URL in the sample app! :smile: